### PR TITLE
Casual friday:  Now on a monday

### PR DESCRIPTION
**Pull request by JStheguy**
-Tacticool Trousers: Camo patterend pants that sometimes spawn in the casual wardrobe.
-Military Jacket: Like an armored vest but covers arms and chest instead of groin and chest.
-Jacket Changes: Regular and Leather jacket both only cover the chest and arms now.
-Red, blue, green, and yellow bandannas now have the same style as the black, skull, and botanics bandannas.

### CODE REVIEW

**Comment by ghost** [comment]
Pics please

**Comment by ghost** [comment]
Remove the bomber jackets from the grey lockers so they're exclusive to the casual lockers

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -435,4 +435,15 @@
 	gender = PLURAL
 	body_parts_covered = GROIN|LEGS
 	fitted = 0
-	can_roll = 0
+	can_roll = 0
+
+/obj/item/clothing/under/tactitrousers
+	name = "tacticool trousers"
+	desc = "A pair of forest camo trousersm, not exactly useful in this enviornment."
```
Comment (Atskadan) [inline code/modules/clothing/under/miscellaneous.dm]: trousersm

**Comment by Psyentific** [comment]
Can you add military jackets to Security wardrobes or lockers? I don't think leaving it to a 'chance' in the HoS Locker is a good way to add it.

**Comment by ghost** [comment]
It's not in a chance, it's in the HoS' alt wardrobe locker since he's got a lovely assortment of clothes now.

I think what you should do is put those security coats (three of them) in security wardrobes (one spawns in sec, one in sec checkpoint, plenty of coats to go around for sec officers) and then put one of those armored military jackets in each sec equipment locker as an alternative from the standard armor. Don't forget to add it to the warden's locker.

Also, move the HoS' security winter coat to his wardrobe locker as well.

To make it simple:

Add one military jacket to the secure security officer locker
Add one military jacket to the secure Warden's locker
Add three security winter coats to the unsecure security wardrobe lockers
Remove the winter coat from the secure security officer locker
Move the security winter coat in the HoS' secure locker to the HoS wardrobe locker (I'm not sure if this is already the case)
Keep the military jacket in the HoS' wardrobe locker, his secure locker is incredibly cluttered and should be reserved for his ultra robust armor (peacoat and jensen trench)

**Comment by ghost** [comment]
:+1:

**Comment by ghost** [comment]
merge when?

**Comment by RemieRichards** [comment]
Can't Auto Merge.
You need to pull changes from the Master branch.

**Comment by ghost** [comment]
ummm yeah i think you need icons/mob/uniform.dmi and both mask files (mob and icon) because i updated those recently with mask flipping.so import your changes to the new files por favor (i wish there was an easier way to do this, but that's the way it is)

i'm not sure how to find conflicts exactly as far as which files can't be auto merged.

**Comment by Atskadan** [comment]
how2conflicts:
pull the new files
see which ones git says are conflicted
fix

**Comment by JStheguy** [comment]
Odd, there is nothing that could be a conflict wit the obj masks.dmi.  The only differences between the two versions is that mine uses a different set of sprites for the bandanas.

**Comment by JStheguy** [comment]
Well, everything seems to be fucking breaking and not working right.
Nuking branch.

**Pull request closed without merging**